ToDo/src/components/Header/index.tsx

```diff
@@ -0,0 +1,37 @@
+import {nanoid} from '@reduxjs/toolkit';
+import React from 'react';
+import {StyleSheet, View} from 'react-native';
+import {useAppDispatch} from '../../hooks/hooks';
+import {addTask} from '../../store/slices/tasksSlice';
+import TaskForm from '../TaskForm';
+
+function Header() {
+  const dispatch = useAppDispatch();
+
+  function onCreate(text: string) {
+    dispatch(
+      addTask({
+        id: nanoid(),
```

Comment:
Let's not keep any business logic inside UI components. Creating ids and setting `isCompleted` should not be responsibility of UI layer. Let's make this action take only `text` and initialize other fields in reducers